docs/references/api.rst

```diff
@@ -1949,6 +1950,66 @@ are:
                "reboot_required": "yes|no|yes-kernel-livepatches-applied"
            }
 
+u.pro.services.dependencies.v1
+========================================
+
+This endpoint will return a full list of all service dependencies,
+regardless of the current system state. That means it will always return
+the same thing until new services are added, or until we add/remove
+dependencies between services.
+
+- Introduced in Ubuntu Pro Client Version: ``32~``
+- Args:
+
+  - This endpoint takes no arguments.
+
+.. tab-set::
+
+   .. tab-item:: Python API interaction
+      :sync: python
+
+      - Calling from Python code:
+
+        .. code-block:: python
+
+           from uaclient.api.u.pro.services.dependencies.v1 import dependencies
+           result = dependencies()
+
+      - Expected return object:
+
+        - ``uaclient.api.u.pro.services.dependencies.v1.DependenciesResult``
+
+          .. list-table::
+             :header-rows: 1
+
+             * - Field Name
+               - Type
+               - Description
+             * - ``services``
+               - *List[ServiceWithDependencies]*
```

Comment:
Does ServiceWithDependencies need to be treated as a single word? The spellchecker doesn't like it very much.